index.html

```diff
@@ -56,6 +56,148 @@ <h2>I'm Cristhian. Glad to see you!</h2>
           </ul>
         </div>
       </section>
+      <section class="section-p2">
```

Comment:
- [ ] kindly consider giving this section a glid display because it offers a grid-based layout system, with rows and columns, making it easier to design web pages and this will make your styling easy and fulfill the below requirement at the same time 
> The main container of the projects has a `display:grid`